buildenv/jenkins/variables/defaults.yml

```diff
@@ -158,6 +158,37 @@ linux_x86-64_cmprssptrs:
       9: 'hw.arch.x86 && sw.os.ubuntu.16'
       10: 'hw.arch.x86 && sw.os.ubuntu.16'
 #========================================#
+# Linux x86 64bits Compressed Pointers /w CMake
+#========================================#
+linux_x86-64_cmprssptrs_cmake:
+  boot_jdk:
+    8: '/usr/lib/jvm/java-7-openjdk-amd64'
+    9: '/usr/lib/jvm/adoptojdk-java-80'
+    10: '/usr/lib/jvm/adoptojdk-java-90'
+  release:
+    8: 'linux-x86_64-normal-server-release'
+    9: 'linux-x86_64-normal-server-release'
+    10: 'linux-x86_64-normal-server-release'
+  freemarker: '/home/jenkins/freemarker.jar'
+  openjdk_reference_repo: '/home/jenkins/openjdk_cache'
+  node_labels:
+    build:
+      8: 'hw.arch.x86 && sw.os.ubuntu.16'
+      9: 'hw.arch.x86 && sw.os.ubuntu.16'
+      10: 'hw.arch.x86 && sw.os.ubuntu.16'
+    test:
+      8: 'hw.arch.x86 && sw.os.ubuntu.16'
+      9: 'hw.arch.x86 && sw.os.ubuntu.16'
+      10: 'hw.arch.x86 && sw.os.ubuntu.16'
+  extra_configure_options:
+    8: '--with-cmake --disable-ddr'
```

Comment:
Why --disable-ddr?

Reply:
the cmake builds currently don't support generating the ddr artifacts